packages/compass-query-bar/src/components/option-editor.tsx

```diff
@@ -143,6 +144,38 @@ const OptionEditor: React.FunctionComponent<OptionEditorProps> = ({
     });
   }, [schemaFields, serverVersion]);
 
+  const onFocus = (editorView: EditorEventCallbackContext, content: string) => {
+    if (hasAutofix && content === '') {
+      editorView.setContent('{}', 1);
+      value = '{}';
```

Comment:
You should really never have modify props like that in the callback, it doesn't affect anything as this doesn't trigger any updates on the component and the next re-render will anyway just pass down a correct prop, but it's a bit of a strange thing to do, so can you explain what the purpose of this is?